assets/src/components/SelectCard.js

```diff
@@ -1,10 +1,19 @@
-import React from 'react'
+/* global fetch */
+import React, { useState } from 'react'
 import { withStyles } from '@material-ui/core/styles'
 import Card from '@material-ui/core/Card'
-import CardActionArea from '@material-ui/core/CardActionArea'
 import CardContent from '@material-ui/core/CardContent'
 import CardMedia from '@material-ui/core/CardMedia'
 import Typography from '@material-ui/core/Typography'
+import { CardActions, IconButton, Snackbar, CircularProgress, Divider, Fab, Link as MUILink } from '@material-ui/core'
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank'
+import CheckBoxIcon from '@material-ui/icons/CheckBox'
+import CloseIcon from '@material-ui/icons/Close'
+import { Link as ReactLink } from 'react-router-dom'
+import { yellow, grey } from '@material-ui/core/colors'
+import SaveIcon from '@material-ui/icons/Save'
```

Comment:
I believe they are still used, it's just that saving appears instantaneous so you don't see the effect.  If saving ever has a visible delay these will be noticeable.

Reply:
Hm... that seems a little strange to me. Might be a little jarring if they only see it for a couple milliseconds and then wonder what it is. I don't know what options we have here. I understand it would be valuable if it did take a while (at that point, it functions as a `Spinner`), but how often does that really happen?